src/nvim/version.c

```diff
@@ -47,37 +47,39 @@ char *version_cflags = "Compilation: " NVIM_VERSION_CFLAGS;
 
 static char *features[] = {
 #ifdef HAVE_ACL
-  "+acl",
+"+acl",
 #else
-  "-acl",
+"-acl",
 #endif
 
 #if (defined(HAVE_ICONV_H) && defined(USE_ICONV)) || defined(DYNAMIC_ICONV)
 # ifdef DYNAMIC_ICONV
-  "+iconv/dyn",
+"+iconv/dyn",
 # else
-  "+iconv",
+"+iconv",
 # endif
 #else
-  "-iconv",
+"-iconv",
 #endif
 
 #ifdef HAVE_JEMALLOC
-  "+jemalloc",
+"+jemalloc",
 #else
-  "-jemalloc",
+"-jemalloc",
 #endif
 
 #ifdef FEAT_TUI
-  "+tui",
+"+tui",
 #else
-  "-tui",
+"-tui",
 #endif
-  NULL
+NULL
 };
 
 // clang-format off
 static const int included_patches[] = {
+  1230,
+  1229,
```

Comment:
It's more important to include these in some commit message as vim-patch tokens:

```
vim-patch:8.0.1229
vim-patch:8.0.1230
```

Because soon version.c update will be automated, and it only looks at commit message tokens.

(Whoever merges this PR can just paste the above into the merge commit)